soccer/Processor.cpp

```diff
@@ -582,6 +608,59 @@ void Processor::run() {
     vision.stop();
 }
 
+void Processor::decodeGeometryPacket(const SSL_GeometryFieldSize* fieldSize) {
+    if (fieldSize->field_lines_size() == 0) {
+        return;
+    }
+
+    const SSL_FieldCicularArc* penalty = nullptr;
+    const SSL_FieldCicularArc* center = nullptr;
+    float displacement = 0.500f;  // default displacment
+
+    for (int i = 0; i < fieldSize->field_arcs().size(); i++) {
+        if (boost::iequals(fieldSize->field_arcs().Get(i).name(), "CenterCircle")) {
```

Comment:
The `name()` method returns a `std::string`, so you can just use `==` here instead of boost.